src/dolphin/setup.ts

```diff
@@ -49,6 +50,11 @@ export default function setupDolphinIpc({ dolphinManager }: { dolphinManager: Do
     return { success: true };
   });
 
+  ipc_getUserFolderPath.main!.handle(async ({ dolphinType }) => {
```

Comment:
```suggestion
  ipc_openDolphinSettingsFolder.main!.handle(async ({ dolphinType }) => {
```
What are your thoughts on making this IPC call in charge of opening the folder too? And then it can switch on the OS and open the appropriate folder. I feel like it's like a extra hoop to jump to request the folder via IPC, and then once again call the IPC to open the folder. Can you foresee any other uses for the FE to get the user folder?

Reply:
yeah that makes more sense